# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/wmpal/hotfix-x11.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

Check for and link libX11, it is used directly and the linker does not
resolve symbols via the indirect libXpm dependency.

--- wmpal-0.6.1.orig/configure.ac	2007-07-05 15:49:15.000000000 +0200
+++ wmpal-0.6.1/configure.ac	2026-09-26 17:08:26.399018063 +0200
@@ -7,6 +7,10 @@ AC_PROG_CC
 AC_PROG_INSTALL
 AC_PATH_X
 
+AC_CHECK_LIB(X11, XSetWMHints,
+  ,
+  AC_MSG_ERROR([libX11 was not found]))
+
 AC_CHECK_LIB(Xpm, XpmCreatePixmapFromData,
   ,
   AC_MSG_ERROR([libXpm was not found"]))
